src/cc/CMakeLists.txt

```diff
@@ -76,6 +81,7 @@ if(ENABLE_CPP_API)
 endif()
 
 if(ENABLE_USDT)
+  list(APPEND bcc_api_headers bcc_usdt.h)
   add_subdirectory(usdt)
   list(APPEND bcc_common_libs_for_a usdt-static)
   list(APPEND bcc_common_libs_for_s usdt-static)
```

Comment:
`bcc-static` has this (`bcc_common_libs_for_s`) linked below which already has `usdt-static` (from `src/cc/usdt/CMakeLists.txt`). Why do we need to have another `bcc_usdt_sources` rule? Maybe we just need the extra header?

Reply:
I tried, but without the bcc_usdt_sources rule, I don't get the bcc_usdt_* symbols in libbcc-loader-static.a and libbcc.a, and my other project doesn't compile.